src/sentry/static/sentry/app/views/projectReleases/releaseLanding.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {t} from 'app/locale';
+import createReactClass from 'create-react-class';
+import {
+  BashCard,
+  Issues,
+  SuggestedAssignees,
+  Emails,
+  Contributors,
+} from 'sentry-dreamy-components';
+import ApiMixin from 'app/mixins/apiMixin';
+import ReleaseLandingCard from 'app/views/projectReleases/releaseLandingCard';
+
+//move to utils
+const cards = [
+  {
+    id: 0,
+    title: t("You Haven't Set Up Releases!"),
+    message: t(
+      'Releases provide additional context, with rich commits, so you know which errors were addressed and which were introduced for the first time'
+    ),
+    component: Contributors,
+  },
+  {
+    id: 1,
+    title: t('Suspect Commits'),
+    message: t(
+      'Sentry suggests which commit caused an issue and who is likely responsible so you can triage'
+    ),
+    component: SuggestedAssignees,
+  },
+  {
+    id: 2,
+    title: t('Release Stats'),
+    message: t(
+      'Set the commits in each release, and which issues were introduced or fixed in the release.'
+    ),
+    component: Issues,
+  },
+  {
+    id: 3,
+    title: t('Easy Resolution'),
+    message: t(
+      'Automatically resolve issues by including the issue number in your commit message.'
+    ),
+    component: BashCard,
+  },
+  {
+    id: 4,
+    title: t('Deploy Emails'),
+    message: t('Receive email notifications when your code gets deployed.'),
+    component: Emails,
+  },
+];
+
+const ReleaseLanding = createReactClass({
```

Comment:
I thought `createReactClass` was on the way out and that ES6 classes should be preferred?

Reply:
ES6 classes are preferred, the exception being with `Reflux` mixins.